### ISSUE TITLE
BUG: case for which run_fg does nothing!

### ISSUE THREAD

**Issue by paugier** [snek5000/snek5000#163]
snek5000-cbox's CI found a funny issue. It's easy to reproduce even with snek5000-tgv:

In a fresh virtual env:

```
pip install --upgrade pip
pip install "snek5000[tests] @ git+https://github.com/snek5000/snek5000.git"
pip install --no-deps --force-reinstall "fluidsim-core @ https://foss.heptapod.net/fluiddyn/fluidsim/-/archive/branch/default/fluidsim-branch-default.tar.gz#subdirectory=lib"
pip install --no-deps --force-reinstall "pymech @ git+https://github.com/eX-Mech/pymech.git"
cd snek5000/docs/examples/snek5000-tgv
pip install -e .
cd ../scripts
python simul_tgv_short.py
```

The directory is created but the simulation is not run. No error are given.

Note that `snakemake -j1 run_fg` in the simulation directory also does nothing...

**Comment by paugier** [comment]
This is our problem in https://github.com/snek5000/snek5000-cbox/pull/27 and https://github.com/snek5000/snek5000-cbox/pull/23

**Comment by paugier** [comment]
BTW

```
$ snakemake -j1 --resources nproc=2 run_fg
not enough values to unpack (expected 2, got 1)
```

I don't understand how to set nproc... And from where comes this error...

**Comment by paugier** [comment]
Other remark, by running snakemake we see the issue with the log being repeated twice (#38).

**Comment by paugier** [comment]
This specific issue seems to be related to the snakemake version. Bug with snakemake 7.18.0 (and 7.18.1), No bug with snakemake 7.17.1.

**Comment by ashwinvis** [comment]
Is this reported in Snakemake's tracker?

**Comment by paugier** [comment]
I can't find any issue looking like this problem. I have no ideas of what happens with snakemake 7.18.0 (which is very recent). I can confirm that a quick fix is to use snakemake 7.17.1.

I think you are the snakemake expert so it would be fine that you have a look.

You could also have a look at the 2 (unrelated) remarks that I made in this thread :-)

**Comment by ashwinvis** [comment]
This seems to work from the command line
```sh
snakemake -j1 --set-resources='run_fg:nproc=2' run_fg
```

**Comment by ashwinvis** [comment]
The equivalent API is `overwrite_resources` keyword argument, which unfortunately is undocumented and cumbersome to use.

See https://github.com/snakemake/snakemake/pull/1041/

**Comment by ashwinvis** [comment]
Fixed by #185 

Important deprecation: `sim.make.exec('run_fg', set_resources={'nproc': 2})` is the new API. Passing `resources` keyword argument is still accepted but emits a deprecation warning.
